Wrapper.h

```diff
@@ -0,0 +1,68 @@
+#pragma once
+#include "Wrapper.h"
+#include "Laundromat.h"
+#include "Restaurant.h"
+#include "Speaker.h"
+#include "Server.h"
+#include "Equipment.h"
+#include "Data.h"
```

Comment:
Don't include what you don't use
forward declare these types once you remove the implementations from this .h file and move them to the cpp file.
Include these headers in the cpp file
- [ ] revise